### ISSUE TITLE
Printing Creates Language Specialty 

### ISSUE THREAD

**Issue by DBuckLee** [chummer5a/chummer5a#1881]
(Please fill out the below as accurately as possible. More detail makes it easier for us to find the problems and fix them.)

### Environment
Chummer Version:  5.194.0
Environment: Windows 7
Runtime:  probably .NET 4.5.1

### Expected behaviour
Native languages should list normally

### Actual behaviour
When printing the native language gains a Speaking Specialty

### Steps to reproduce the behaviour
Print character that has a native language

### Link to affected character file
Chummer: https://drive.google.com/open?id=0BxyMFiiPIKWIb2U3LW9ka3hrajQ
PDF:https://drive.google.com/open?id=0BxyMFiiPIKWIc3ZVbHpXaUtTMTg

**Comment by Lochabar** [comment]
There is a speciality present in the character file

        <specs>
          <spec>
            <guid>c43552a2-f3f4-4d4e-a037-8c3d6164818f</guid>
            <name>Speak</name>
          </spec>
        </specs>

Are you saying this was not there before?

**Comment by DBuckLee** [comment]
If the rating is dropped to 0  (Native) then the specialization should not
show, correct?   This is how it works in the file, but it appears when
printed.

On a separate note, it looks like the knowledge skills are using active
skill points instead of knowledge skills points too.  I am not sure why
this is happening.  I noticed this when I added a rating to any of the
current knowledge skills.

  Thank you,
 Dan
